src/main/java/de/retest/recheck/FileOutputFormat.java

```diff
@@ -0,0 +1,10 @@
+package de.retest.recheck;
```

Comment:
Shouldn't this be in the persistence package?

Reply:
It wasn't before, but I can move it.